lib/loader.js

```diff
@@ -11,6 +11,7 @@ module.exports = app => {
     directory: dirs,
     target: app.io.middleware,
     inject: app,
+    typescript: true,
```

Comment:
This property has deprecated

Reply:
Loader load ts file by env( see https://github.com/eggjs/egg-core/blob/4d4113cfd27d1e8ce4ce65d2d19b0035b5291dcc/lib/loader/file_loader.js#L125 ), this env will be set if you are using `egg-bin dev --ts`

Reply:
`typescript: true` is not necessary here.

Reply:
Well……Just for a confirm：The problem 'https://github.com/eggjs/egg/issues/2398#issuecomment-386264963' is really fixed? If your answer is YES, and I'll remove this :)

Reply:
@Maledong Yes

Reply:
Thanks! I'll remove and resubmit:)